### DISCUSSION TITLE
Cant install DBEAVER exe on windows 10

### DISCUSSION THREAD

**Discussion by ashokSQLtechi** [dbeaver/dbeaver#13847] [Q&A]
- Operating system (distribution) and version: windows 10
- DBeaver version: 21.1.0
- Java version 11
- Additional extensions

![image](https://user-images.githubusercontent.com/88874135/129312152-792a7a6c-0c73-463c-ac1f-67192b8d656f.png)

#### Describe the problem you're observing:

when i tried to install windows enterprise edition it shows installing but never complete.

**Comment by ashokSQLtechi** [comment]
plz suggest how to fix this error.

**Comment by serge-rider** [comment]
Maybe the installer was corrupted during download. Or some local software blocks further installation.
Did you try to download another version? (e.g. 21.0)
